velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -25,76 +25,85 @@ namespace {
 static constexpr char kWomanFacepalmingLightSkinTone[] =
     "\xF0\x9F\xA4\xA6\xF0\x9F\x8F\xBB\xE2\x80\x8D\xE2\x99\x80\xEF\xB8\x8F";
 
-class StringTest : public SparkFunctionBaseTest {
+class StringTest : public test::FunctionBaseTest {
  protected:
+  static void SetUpTestCase() {
+    parse::registerTypeResolver();
+    // Register Spark functions with prefix.
+    sparksql::registerFunctions(prefix_);
+    memory::MemoryManager::testingSetInstance({});
```

Comment:
This is to fix https://github.com/facebookincubator/velox/pull/7377 by registering Spark functions with a prefix to distinguish from Presto ones. I'm not sure why this issue only occurs in this PR.